medium.go

```diff
@@ -143,6 +146,7 @@ type Medium struct {
 	ApplicationSecret string
 	AccessToken       string
 	Host              string
+	Timeout           int
```

Comment:
Yes, sure. Being explicit on the type makes the API more flexible. Updating the PR shortly.